fvm/transactionEnv.go

```diff
@@ -953,6 +953,22 @@ func (e *TransactionEnv) ValueDecoded(duration time.Duration) {
 	e.metrics.ValueDecoded(duration)
 }
 
+func (m *TransactionEnv) BLSVerifyPOP(_ *runtime.PublicKey, _ []byte) (bool, error) {
+	panic("unexpected BLSVerifyPOP call")
```

Comment:
I think a better way might be returning this error instead of panic (then we don't have to go through recovery, etc, ...)
e.g. return errors.NewOperationNotSupportedError("BLSVerifyPOP")

Reply:
Ah this was left over from what I was basing this on Cadence master, it shouldn't be necessary anymore.